### PR TITLE
cat commnads are replace by reading from file

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/commons#289

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/commons/objects/node_context/__init__.py [hunk 1]
```diff
@@ -29,9 +30,17 @@ def __init__(self, machine_id=None, node_id=None, fqdn=None,
...
-        return str(out)
+        try:
+            out = None 
+            with open('/etc/machine-id', 'r') as f:
+                out = f.read().strip('\n') 
+        except IOError as ex:
+            exc_type, exc_value, exc_tb = sys.exc_info()
+            traceback.print_exception(
+                exc_type, exc_value, exc_tb, file=sys.stderr)
+            sys.stderr.write(
```
Comment (GowthamShanmugam) [inline tendrl/commons/objects/node_context/__init__.py]: Message class required node_context to log the message, so we can't log this exception using message class

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @nthomas-redhat  please review

File: tendrl/commons/objects/node_context/__init__.py (hunk 1 above)
Comment (nthomas-redhat) [inline tendrl/commons/objects/node_context/__init__.py]: what is the point here? Ideally you should log using the Event(https://github.com/Tendrl/commons/pull/297/commits/80bc98313d23b8c9afc45bf0404ea9ef0c7d52b0#diff-e3ba3b401bfdec0d0df1d4ee946980abR34)?

Comment (GowthamShanmugam) [inline tendrl/commons/objects/node_context/__init__.py]: @nthomas-redhat problem is message class needs node_context to log a message using event. But here still node_context is not available in NS, so we can't log the message using message class.

**Comment by GowthamShanmugam** [comment]
@nthomas-redhat no i am not blacklist cat, i will do it

**Comment by GowthamShanmugam** [comment]
@nthomas-redhat i have black listed cat command

File: tendrl/commons/objects/node_context/__init__.py (hunk 1 above)
Comment (GowthamShanmugam) [inline tendrl/commons/objects/node_context/__init__.py]: @nthomas-redhat if you see in message class,for each message it assigning node_id automatically,
self.node_id = NS.node_context.node_id
so at this time node id not created so message class wil give error, so we cant use message class for this exception

File: tendrl/commons/utils/cmd_utils.py [hunk 2]
```diff
@@ -9,7 +9,6 @@
 
 SAFE_COMMAND_LIST = [
     "lsblk",
-    "cat",
```
Comment (nthomas-redhat) [inline tendrl/commons/utils/cmd_utils.py]: Check all our source and make sure we are not using 'cat' anywhere?
Please confirm

Comment (GowthamShanmugam) [inline tendrl/commons/utils/cmd_utils.py]: ok

File: tendrl/commons/objects/memory/__init__.py [hunk 3]
```diff
@@ -24,10 +26,20 @@ def _getNodeMemory(self):
                    "Type":      "type"}, ...], ...}
 
         '''
-
-        cmd = cmd_utils.Command("cat /proc/meminfo")
-        out, err, rc = cmd.run()
-        out = str(out)
+        try:
+            out = None
+            with open('/proc/meminfo', 'r') as f:
```
Comment (r0h4n) [inline tendrl/commons/objects/memory/__init__.py]: You dont need 'r' here, it is the default mode

Comment (GowthamShanmugam) [inline tendrl/commons/objects/memory/__init__.py]: ok i will change it

**Comment by GowthamShanmugam** [comment]
@r0h4n changes done

**Pull request merged**